### ISSUE TITLE
Update expiration date from MM/YYYY to MM/YY

### ISSUE THREAD

**Issue by smarziano** [smartchicago/kimball#176]
Currently on staging, the input for gift card expiration dates is MM/YYYY. I want to change this to match the date format on gift cards to MM/YY.

**Comment by smarziano** [comment]
This has been updated on staging... waiting on merge.

**Comment by smarziano** [comment]
This is live. We have a new issue #203 for consistency across pages, but this issue has been resolved. Closing.
